containers/Makefile

```diff
@@ -164,28 +160,27 @@ build/%: %
 
 # Templates
 build/%: %.j2 build/config-with-versions.yml ../tools/render-jinja-template.sh
-	../tools/render-jinja-template.sh         \
-		build/config-with-versions.yml        \
-		"$<"                                  \
+	../tools/render-jinja-template.sh\
+		build/config-with-versions.yml \
+		"$<"                           \
 		"$@"
 
 build/config-with-versions.yml: versions.yml config.yml ../tools/merge-yaml.py
-	../tools/merge-yaml.py                     \
-		"$<"                                   \
-		"config.yml"                           \
+	../tools/merge-yaml.py\
+		"$<"                \
+		"config.yml"        \
 		--output "$@"
 
-
```

Comment:
Please avoid unnecessary changes, like removing these empty lines.

I'm not saying that removing them is a bad idea but I don't want you and @bartoszbetka to get into a loop where you remove them and he adds them back :)

When editing an existing file please adapt to the style already used in it unless the issue is specifically about refactoring the style.

Reply:
It was the only one occurrence of two empty lines in a row, so it looked more than correcting a mistake than imposing own coding style. Fully agree about adapting to existing style.